packaging/fedora/snapd.spec

```diff
@@ -97,7 +97,7 @@
 
 Name:           snapd
 Version:        2.35.1
-Release:        0%{?dist}
+Release:        1%{?dist}
```

Comment:
Please leave this at `0%{?dist}`. This is intentional.

Reply:
Do I remove the change log entry completely then? Is it going to be changed when you release it (along with other stuff) else and be released as 2.35.1-1 still? Otherwise I don't understand how it's going to be work.

Reply:
`Release:` is set to `0%{?dist}` upstream, and I change it accordingly downstream in Fedora.

Changelog entries should also not be added to the upstream spec as such, since they'll get generated when we make a release by @mvo5.